cli/trigger_check.go

```diff
@@ -163,18 +183,22 @@ func triggerCheck(c *cli.Context) error {
 
 	//
 	// Main
-	isPRMode, err := isPRMode(prGlobalFlagPtr, inventoryEnvironments)
-	if err != nil {
-		registerFatal(fmt.Sprintf("Failed to check  PR mode, err: %s", err), warnings, triggerParams.Format)
-	}
-
-	workflowToRunID, err := getWorkflowIDByParamsInCompatibleMode(bitriseConfig.TriggerMap, triggerParams, isPRMode)
+	triggerModel := map[string]string{}
+	pipelineToRunID, err := getPipelineIDByParams(bitriseConfig.TriggerMap, triggerParams)
```

Comment:
Do the pipeline check in the original method instead (and rename it). Return both pipeline ID & workflow ID.